src/jmcomic/jm_client_impl.py

```
@@ -709,6 +709,9 @@ def fetch_detail_entity(self, jmid, clazz):
             })
         )
 
+        if resp.res_data.get('name') is None:
+            ExceptionTool.raise_missing(resp, jmid)
```

Ruff (0.8.2)

`ExceptionTool` may be undefined, or defined from star imports

(F405)